WebRoot/js/zimbraMail/abook/view/ZmGroupView.js

```diff
@@ -554,7 +554,7 @@ function(x, y, width, height) {
 	if(this._addNewField){
 		Dwt.setSize(this._addNewField, Dwt.DEFAULT, 100);
 	}
-	var memberListHeight = height - 50;
+	var memberListHeight = height - 150; //set member list height to totalContainerHeight - 150, to make sure that elements are of atleast enough required height
```

Comment:
`height` is actually container height i.e height of full `Create Group/DL` page. So practically I don't see height to be less that 150px